SPECS/node-problem-detector/node-problem-detector.spec

```diff
@@ -110,7 +110,7 @@ make test
 - Bump release to rebuild with golang 1.18.3
 
 * Wed Jun 01 2022 Olivia Crain <oliviacrain@microsoft.com> - 0.8.10-2
-- Add explicit check/run-time dependencies on mariner-release
+- Add explicit check/run-time dependencies on azurelinux-release
```

Comment:
Similar comment about the changelog: I think we shouldn't change the past.:)

Reply:
Updated